Description: Increade buffer size in python test
 GnuPG >= 2.5.10 adds additional version info to signatures, rendering the
 previously used buffer size too small.
Author: Meik Michalke <meik.michalke@gnupg.com>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lang/python/tests/t-idiomatic.py
+++ b/lang/python/tests/t-idiomatic.py
@@ -79,7 +79,7 @@ if sys.version_info[0] == 3:
     bio.truncate(1)
     if len(bio.getvalue()) != 1:
         # This version of Python is affected, preallocate buffer.
-        preallocate = 128 * b'\x00'
+        preallocate = 256 * b'\x00'
     else:
         preallocate = b''
 
